src/code/z_construct.c

```diff
@@ -1,4 +1,5 @@
 #include "global.h"
 
-void func_801AAAA0(PlayState* play) {
+// Remnant from OoT
+void Construct_InitRegs(PlayState* play) {
 }
```

Comment:
would prob be good to wait for oot to decide on a name for this, and then name it accordingly. 
I for example am not sure of "Construct" being the system name for it, even tho it is the name of the file. Seems like something that will need more discussion on that end

Reply:
if this is the function I think it is, https://github.com/zeldaret/oot/pull/1354 (not merged) renames it from `func_80112098` to `Regs_InitData`